db/seeds.rb

```diff
@@ -1,7 +1,22 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+
+
+favorite_drinks = [
+    {
+      user_id: 2,
```

Comment:
the seed file would be more resilient if the id's for users are not hardcoded and it seems dependent on users being created in the db already. Perhaps create some users in the db within the seed file and reference those users.

Also let's say someone wants to seed their db twice just to have more data to work with. The favorite drinks in this case would always be associated with users with ids 2 and 3 and not the newly created users in the seed file

Reply:
looks like building logic to seed drinks here will also be needed